<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompletionSettingsPage</class>
 <widget class="QWidget" name="CompletionSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>244</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="caseSensitivityLabel">
        <property name="text">
         <string>&amp;Case-sensitivity:</string>
        </property>
        <property name="buddy">
         <cstring>caseSensitivity</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="caseSensitivity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Full</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>First Letter</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="completionTriggerLabel">
        <property name="text">
         <string>Activate completion:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QComboBox" name="completionTrigger">
        <item>
         <property name="text">
          <string>Manually</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>When Triggered</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="autoInsertBrackets">
        <property name="toolTip">
         <string>Automatically insert brackets and semicolons when appropriate.</string>
        </property>
        <property name="text">
         <string>&amp;Automatically insert brackets</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="spaceAfterFunctionName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Insert &amp;space after function name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="partiallyComplete">
        <property name="toolTip">
         <string>Insert the common prefix of available completion items.</string>
        </property>
        <property name="text">
         <string>Autocomplete common &amp;prefix</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>autoInsertBrackets</sender>
   <signal>toggled(bool)</signal>
   <receiver>spaceAfterFunctionName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
